e2e-tests/README.md

```diff
@@ -49,7 +49,11 @@ To specify your own repository for the Percona XtraDB Cluster Operator docker im
 ```
 export IMAGE=bob/my_repository_for_test_images:K8SPXC-622-fix-feature-X
 ```
+If you use Apple Silicon (or another ARM64 based architecture) and plan to start operator on amd64 (e.g. AWS EC2, ECS, etc.) please specify docker platform:
+```
+ export DOCKER_DEFAULT_PLATFORM=linux/amd64
```

Comment:
I think we should set env by default in the build script.
and user can change var only when non-`linux/amd64` platform needed

Reply:
Could you please give a bit details why to use export is better than use docker flag? @delgod